src/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
what does this file do?  Specially the theme_color and background_color.. where do they help.

Reply:
Caching using Service Wrokers is one of the things that comes with Progressive Web App paradigm. This file takes care of things like icon set for different devices, background color for splash screen, browser top toolbar color etc.

Reply:
cool